package.json

```diff
@@ -11,7 +11,10 @@
     "lint": "eslint --ext .ts src/ test/",
     "test": "jest --forceExit",
     "format": "yarn format:check --write",
-    "format:check": "prettier --check \"**/*.{md,json}\" \"{src,test}/**/*.ts\""
+    "format:check": "prettier --check \"**/*.{md,json}\" \"{src,test}/**/*.ts\"",
+    "prepack": "run-s build",
```

Comment:
`run-s` is provided by package `npm-run-all`, you should add it to `devDependencies`